charts/incubator/eco/values.yaml

```diff
@@ -84,14 +72,11 @@ persistence:
     enabled: true
     mountPath: /serverdata/serverfiles
   network-config:
-    enabled: true
```

Comment:
This is needed afaik, unless we default to true!

Reply:
whoops